ansible/roles/whiskhost/tasks/deploy.yml

```diff
@@ -2,6 +2,6 @@
 # Add user and patch kernel.
 
 - include: addUser.yml
-  when: "'prod' in deployment"
+  when: docker.user is defined
```

Comment:
No documentation needed in open source code, as user is not expected to do anything